src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1734,7 +1734,6 @@
   </Node>
 
   <!-- Node only used during nullability flow analysis to represent an expression with an updated nullability -->
-  <!-- PROTOTYPE(NullableReferenceTypes): Derive from BoundValuePlaceholderBase. -->
```

Comment:
Cannot derive from `BoundValuePlaceholderBase` since that type requires non-null `Type`.